docs/api.md

```diff
@@ -342,14 +349,6 @@ with_request_url("/users/42") do
 end
 ```
 
-To use a specific host, pass the host param:
-
-```ruby
-with_request_url("/users/42", host: "app.example.com") do
-  render_inline(MyComponent.new)
-end
-```
```

Comment:
It looks like this may have been inadvertently deleted by regenerating the API docs, which happened because it seems this part was manually added directly in `api.md` in https://github.com/ViewComponent/view_component/pull/1773, rather than in the corresponding YARD comment.

Reply:
I've opened a PR here to restore this example here: https://github.com/ViewComponent/view_component/pull/1794